Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use num_bigint::{BigUint, ToBigUint};
+use std::io;
+
+// convert input string to BigUint
+fn read_parse(inp: String) -> BigUint {
+    let age: BigUint = inp.trim().parse().expect("Not a positive integer.");
```

Comment:
This function should return a `Result` rather than crash. Panicing is ok eg. for smaller applications, but it should be done at the outemost layer practical (here `main`).